bot/exts/utils/snekbox.py

```diff
@@ -22,17 +22,87 @@
 
 ESCAPE_REGEX = re.compile("[`\u202E\u200B]{3,}")
 
+# The timeit command should only output the very last line, so all other output should be suppressed.
+# This will be used as the setup code along with any setup code provided.
+TIMEIT_SETUP_WRAPPER = """
+import atexit
+import sys
+from collections import deque
+
+if not hasattr(sys, "_setup_finished"):
+    class Writer(deque):
+        def __init__(self):
+            super().__init__(maxlen=1)
+
+        def write(self, string):
+            if string.strip():
+                self.append(string)
+
+        def read(self):
+            return self.pop()
+
+        def flush(self):
+            pass
+
+    sys.stdout = Writer()
+
+    def print_last_line():
+        if sys.stdout:
```

Comment:
I don't understand this, when would `sys.stdout` be falsely?

Reply:
If the queue is empty.